src/core.jl

```diff
@@ -122,20 +131,26 @@ isapprox(n::Node, f::Node) = unbox(n) ≈ unbox(f)
 zero(n::Node) = zero(unbox(n))
 one(n::Node) = one(unbox(n))
 
+# Let the user get the `size` and `length` of `Node`s.
```

Comment:
I diddn't add this, I just moved them here from [`sensitivities/array.jl`](https://github.com/invenia/Nabla.jl/pull/189/files#diff-bb6d794c031dab7ad70c30e18604e3088568fdfdeb07788e66b8438aac537fceL4-L5)
which has now been deleted.
They shouldn't really have been there to begin with, since they are not sensitivies except in the sense of saying "there is no sensitivity"